src/pybind/mgr/dashboard/frontend/src/app/shared/components/notifications-sidebar/notifications-sidebar.component.scss

```diff
@@ -66,3 +66,8 @@ hr {
 .card-text {
   margin-right: 15px;
 }
+
+.mute {
+  margin-right: -17px;
+  margin-top: -4px;
+}
```

Comment:
we could remove this by adding the above bootstrap classes and that way we probably don't need to worry about this in downstream branding.


```suggestion
```